scripts/variables.sh

```diff
@@ -4,34 +4,21 @@
 # Setting date variables
 # **********************
 printf "\nSetting Variables\n"
-export script_dir="${TRAVIS_BUILD_DIR}/scripts"
-export testdir="${TRAVIS_BUILD_DIR}/test_data"
+
+# First setup the working dir
+pushd . > /dev/null
+SCRIPT_PATH="${BASH_SOURCE[0]}";
+if ([ -h "${SCRIPT_PATH}" ]) then
+  while([ -h "${SCRIPT_PATH}" ]) do cd `dirname "$SCRIPT_PATH"`; SCRIPT_PATH=`readlink "${SCRIPT_PATH}"`; done
+fi
+cd `dirname ${SCRIPT_PATH}.` > /dev/null
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/gh/PyFunceble-Templates/MariaDB-Import-Export/pullRequest?prid=4942241)